pkg/machine/shim/networking.go

```diff
@@ -101,6 +102,8 @@ func startNetworking(mc *vmconfigs.MachineConfig, provider vmconfigs.VMProvider)
 	}
 
 	c := cmd.Cmd(binary)
+
+	logrus.Debugf("gvproxy command-line: %s %s", binary, strings.Join(cmd.ToCmdline(), " "))
```

Comment:
Debug needs to be removed later

Reply:
I actually think this would be useful information as long as its something that isn't spit out by gvproxy already

Reply:
Ah I read this as error and not debug, my bad.